docs/howto/access-data.md

```diff
@@ -0,0 +1,45 @@
+### **Accessing the data**
+
+**Option 1: Using the data links**
+
+If we already have a workflow in place for downloading our data, we can use *earthaccess* as a search-only library and get HTTP links from our query results. This could be the case if our current workflow uses a different language and we only need the links as input.
+
+```python
+
+# if the data set is cloud hosted there will be S3 links available. The access parameter accepts "direct" or "external", direct access is only possible if you are in the us-west-2 region in the cloud.
```

Comment:
```suggestion
# if the data set is cloud-hosted there will be S3 links available. The access parameter accepts "direct" or "external". Direct access is only possible if you are in the us-west-2 region in the cloud.
```

Reply:
Does this mean that `direct` will return s3 links? It's a bit confusing to have references to s3 and direct access when the guidance above only talks about downloading data with HTTP links.